src/rdf4cpp/BigDecimal.hpp

```diff
@@ -435,6 +435,25 @@ struct BigDecimal {
         return false;
     }
 
+    static double cast_unscaled_to_double_safe(UnscaledValue_t const &value) noexcept {
+        // There is a bug in boost versions <1.86.0
+        // that can cause a crash in the cast `static_cast<double>(cpp_int)`
+        // https://github.com/boostorg/multiprecision/issues/553
+
+#if BOOST_VERSION < 108600
+        // workaround from https://github.com/scylladb/scylladb/commit/51d09e6a2a407ea9b9ad380ba30e5558a25bb8be#diff-81e6758bbedbe566a51706c5af1ea68be225c36feb68983c3de0a69138f01264R73
+        static auto const min = UnscaledValue_t{std::numeric_limits<double>::lowest()};
```

Comment:
the workaround in the linked fix is for `utils::multiprecision_int`. The default value for `UnscaledValue_t` is `cpp_int`. Could there be an issue if we use something else (do we actually use another type?)

Reply:
I didn't double check it but I was assuming that `utils::multiprecision_int` is `boost::multiprecision::cpp_int`. 

At least this workaround works the way I've written it (tried it with an older boost ver)